src/views/admin/PersonManagementView.vue

```diff
@@ -80,24 +155,59 @@
         : '---';
       return {
         ...person,
-        rollen: rollen,
+        rollen: rollenZuordnungen,
         administrationsebenen: administrationsebenen,
-        klassen: klassen,
+        klassen: klassenZuordnungen,
         person: { ...person.person, personalnummer: personalnummer },
       };
     });
   });
 
   const handleSearchFilter = (filter: string): void => {
     searchFilter.value = filter;
-    personStore.getAllPersons(searchFilter.value);
+    applySearchAndFilters();
   };
 
+  /* Watcher to detect when the search input for Schulen is triggered.
+     Always request from backend to keep display of hits consistent */
+  watch(searchInputSchulen, async (newValue: string, _oldValue: string) => {
+    organisationStore.getAllOrganisationen({ searchString: newValue, includeTyp: OrganisationsTyp.Schule });
+  });
+
+  /* Watcher to detect when the search input for Rollen is triggered.
+     Always request from backend to keep display of hits consistent */
+  watch(searchInputRollen, async (newValue: string, _oldValue: string) => {
```

Comment:
Wäre es einfacher statt des Watchers, die aktualisierung an das @input event des Eingabefelds zu hängen?

Reply:
v-autocomplete has update:search event, which we now use instead